src/main/java/org/apache/commons/crypto/utils/NativeCodeLoader.java

```diff
@@ -222,10 +222,10 @@ private static File extractLibraryFile(String libFolderForCurrentOS,
    */
   public static String getVersion() {
     URL versionFile = NativeCodeLoader.class
-        .getResource("/META-INF/maven/com.intel.chimera/chimera/pom.properties");
+        .getResource("/META-INF/maven/org.apache.commons.crypto/chimera/pom.properties");
```

Comment:
This will be addressed in [CRYPTO-10](https://issues.apache.org/jira/browse/CRYPTO-10).